packages/app/src/app/pages/Sandbox/Editor/Workspace/ConnectionNotice/ConnectionNotice.js

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react';
```

Comment:
Would it be better to call this file `index.js` for consistency with the other files?

Reply:
No, the reason why we always split `index.js` and `Component.js` is to deliberately improve readability in a contributor's code editor. It quickly becomes confusing which file you're looking at when every component is defined inside of an `index.js`. Additionally, we may want to export multiple components, functions, etc from a single directory using named exports. A good example of this is a component that comes paired with say a GraphQL query fragment. It makes sense to keep those files separate for readability's sake, much like how we split component definitions from css-in-js with `elements.js`. The goal is to break files down into small, highly readable chunks. My general philosophy is that if I have to scroll to see all the contents of a single file, it's getting too big. Sometimes that's unavoidable, but you can use your best judgement there.

Reply:
Right, the only thing I'm afraid of is that we're now following two different naming schemes inside the repo (alongside two languages). For new contributors we need to be clear what the preferred way is.